server/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const path = require('path');
+
+const app = express();
+
+const port = 3000;
+
+app.use(express.static(path.join(__dirname, '../public')));
```

Comment:
you probably want to load the static files at '/location/:locationId' instead of just at '/'.

Reply:
This is fixed in the next branch that I will merge